README.md

```diff
@@ -190,6 +190,7 @@ Table of Contents
   * [OpenAPI3 Designer](https://interaction-design.co.za/openapidesigner/) — Visually create Open API 3 definitions for free
   * [parsehub.com](https://parsehub.com/) — Extract data from dynamic sites, turn dynamic websites into APIs, 5 projects free
   * [Pixela](https://pixe.la/) - Free daystream database service. All operations are performed by API. Visualization with heat maps and line graphs is also possible.
+  * [Pizzly](https://github.com/bearer/pizzly/) - Simplest, fatest way to integrate an app with an OAuth API.
```

Comment:
whats the SaaS offering here?

Reply:
Hi @ripienaar. It's an open-source project that makes it easy to integrate an application (SPA) with an OAuth API.

Reply:
Yes, but this list is for SaaS offerings, so I am curious what the SaaS angle is here?

Reply:
Pizzly is a self-hosted SaaS. You deploy it on Heroku for example and then you define which API you want to use and _voilà_. You're ready to use it from your frontend or backend. [Here are a few screenshots of the dashboard (i.e. the SaaS)](https://github.com/Bearer/Pizzly/wiki/Reference-:-Dashboard).

Reply:
There's no self-hosted saas, sorry. not suited to the list.